delta/plugins/archive/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/archive/RemoteContextResolutionFixture.scala

```diff
@@ -10,15 +10,15 @@ import ch.epfl.bluebrain.nexus.testkit.IOValues
 trait RemoteContextResolutionFixture extends IOValues {
   implicit private val cl: ClassLoader = getClass.getClassLoader
 
-  implicit val rcr: RemoteContextResolution = RemoteContextResolution.fixed(
-    storageContexts.storages         -> ContextValue.fromFile("contexts/storages.json").accepted,
-    storageContexts.storagesMetadata -> ContextValue.fromFile("contexts/storages-metadata.json").accepted,
-    fileContexts.files               -> ContextValue.fromFile("contexts/files.json").accepted,
-    contexts.archives                -> ContextValue.fromFile("contexts/archives.json").accepted,
-    contexts.archivesMetadata        -> ContextValue.fromFile("contexts/archives-metadata.json").accepted,
-    Vocabulary.contexts.metadata     -> ContextValue.fromFile("contexts/metadata.json").accepted,
-    Vocabulary.contexts.error        -> ContextValue.fromFile("contexts/error.json").accepted,
-    Vocabulary.contexts.tags         -> ContextValue.fromFile("contexts/tags.json").accepted
+  implicit val rcr: RemoteContextResolution = RemoteContextResolution.fixedIO(
```

Comment:
How we load them in the tests is something we will have to improve at some point